airflow/cli/cli_parser.py

```diff
@@ -1381,6 +1388,18 @@ class GroupCommand(NamedTuple):
         func=lazy_load_command('airflow.cli.commands.role_command.roles_create'),
         args=(ARG_ROLES, ARG_VERBOSE),
     ),
+    ActionCommand(
+        name='export',
+        help='Export roles (without permissions) from db to JSON file',
+        func=lazy_load_command('airflow.cli.commands.role_command.roles_export'),
+        args=(ARG_ROLE_EXPORT, ARG_ROLE_EXPORT_FMT, ARG_VERBOSE),
+    ),
+    ActionCommand(
+        name='import',
+        help='Import roles from JSON file to db',
```

Comment:
```suggestion
        help='Import roles (without permissions) from JSON file to db',
```

Sorry, I meant here as well.